src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -402,6 +402,7 @@ private BoundExpression BindExpressionInternal(ExpressionSyntax node, Diagnostic
                 case SyntaxKind.SubtractExpression:
                 case SyntaxKind.DivideExpression:
                 case SyntaxKind.ModuloExpression:
+                case SyntaxKind.RangeExpression:
```

Comment:
Consider calling this ExclusiveRange.  I imagine that hte language could add an InclusiveRange operator in the future as well.

Reply:
I believe this is intended to be a closed (inclusive) interval.

---
In reply to: [152133041](https://github.com/dotnet/roslyn/pull/23312#discussion_r152133041) [](ancestors = 152133041)

Reply:
This is why I left it ambiguous: because we haven't decided yet what it will be. Renaming is simple enough.